docs/content/Dialog2.mdx

```diff
@@ -180,38 +180,39 @@ render(<ShorthandExample2 />)
 
 ## AlertDialog
 
-An `AlertDialog` is a special kind of dialog with more rigid behavior. It is used to alert or confirm a single user action. AlertDialogs can only have at most one button: no buttons or one to confirm the alert dialog. No custom rendering capabilities are provided for AlertDialog.
+AlertDialog is a special kind of dialog with an urgent interruption, and informs the user about a situation.
+They summarize a **single** decision that requires acknowledgement and can only be dismissed by the alert close button or `escape` key.
+Use a clear question or statement for the alert title with an explanation in the content area, such as "Erase Disk storage?".
```

Comment:
This is a bit nitpicky but isn't `Erase Disk storage` a confirmation message that we would use a Confirmation Dialog for?

Reply:
I think we can avoid having to add `AlertDialog` right now by rewording the dialog message in Memex (like @vdepizzol suggested https://github.com/primer/react/pull/1378#issuecomment-901351218)

Reply:
@cmwinters thoughts?